jwst/saturation/x_irs2.py

```diff
@@ -30,37 +30,41 @@
 
 
 def _get_irs2_parameters(input_model, n=None, r=None):
-    """Get the parameters describing IRS2 readout format.
+    """
+    Get the parameters describing IRS2 readout format.
 
     Parameters
     ----------
-    input_model: most likely a RampModel object
+    input_model : DataModel
+        This model is most likely a RampModel object
```

Comment:
Why "most likely"?  Is this always a RampModel? Specifically what model types are ok here?

Reply:
Not sure if what I changed it to is  clear enough. I just pulled in information from the top of the code. 

This is what I made it:
input_model : DataModel or numpy.ndarray.
        Usually it is a RampModel but it can be an  ndarray. 
        If it is an ndarray parameters will be
        assigned default values, unless specified explicitly.
        This is used for getting the width of the reference output and
        the values of NRS_NORM and NRS_REF.

Reply:
Looking more closely at this, it appears this is only used to get two metadata attributes.  If an array (or other type) were passed in, it would still work but just not do anything.  I know this is just style changes, so I'm not suggesting doing this necessarily, but IMO `input_model` could be made optional.  Passing an array in here seems unnecessarily confusing and I'd suggest avoiding it if possible.

For the present, I think it's fine to remove current Line 39 and just make the docstring say something like, "Model from which to retrieve the width of the reference output and the values of NRS_NORM and NRS_REF" or similar

Reply:
One more small note - I'm glad you specified that this can also be a `np.ndarray`, but it might be good to write in the parameter description that if an `ndarray` is passed in, it is ignored.